core/templates/pages/topic-viewer-page/subtopics-list/subtopics-list.directive.html

```diff
@@ -18,9 +18,9 @@
        ng-swipe-left="scroll(false)">
     <div class="oppia-topic-viewer-carousel-tiles">
       <md-card ng-repeat="subtopic in subtopics" class="oppia-subtopic-summary-tile">
-        <subtopic-summary-tile skill-count="subtopic.skill_ids.length"
-                               subtopic-id="subtopic.id"
-                               subtopic-title="subtopic.title"
+        <subtopic-summary-tile skill-count="subtopic.getSkillSummaries().length"
```

Comment:
getSkillSummaries will return skill_ids? (Just wanted to double-check)

Reply:
Yes, as per SubtopicObjectFactory.ts